xds/src/test/java/io/grpc/xds/XdsLoadBalancerProviderTest.java

```diff
@@ -173,7 +173,6 @@ public void parseLoadBalancingConfigPolicy() throws Exception {
     assertThat(configOrError.getConfig()).isInstanceOf(XdsConfig.class);
     assertThat(configOrError.getConfig()).isEqualTo(
         new XdsConfig(
-            "dns:///balancer.example.com:8080",
```

Comment:
There are a bunch of other places in this file that involves `balancerName`, you might want to clean them up as well.

Reply:
Done.